doc/model.rst

```diff
@@ -0,0 +1,683 @@
+current working directory by default if no `path` argument is supplied. When a Model
+instance is passed to ``Experiment.generate()``, a directory within the Experiment directory
+is automatically created to store input and output files from the model.
+
+----------------
+A Standard Model
+----------------
+For standard model deployment, the model runs
+on separate compute nodes from SmartSim orchestrators.
+Standard models are able to communicate with clustered or standalone
+databases. SmartRedis clients connect to a clustered Orchestrator from
+within the Models executable script
+and travel off the simulation compute node to the database compute node to send/retrieve
+data. To use a database within the standard model workload, a
+SmartSim Orchestrator must be launched prior to the start of the Model.
+Standard models are ideal for simulations benefiting from
+distributed computing capabilities, enabling efficient
+parallelized execution across multiple compute nodes.
+
+In the `Instructions` subsection, we provide an example illustrating the deployment of a standard model.
+
+Instructions
+------------
+This example provides a demonstration of how to initialize and launch a standard Model
+within the Experiment workflow.
+
+All workflow entities are initialized through the Experiment API. To create a Model,
+the function ``Experiment.create_model()`` is used. To create Model, we must first
+initialize an instance of the Experiment class. Import the SmartSim Experiment module
+and initialize an Experiment instance. Provide a `name` and `launcher`:
+
+.. code-block:: python
+
+    from smartsim import Experiment
+
+    # Init Experiment and specify to launch locally
+    exp = Experiment(name="getting-started", launcher="local")
+
+Models require run settings objects which are initialized through an Experiment instance.
+Use the experiment instance, `exp`, to access the helper method ``Experiment.create_run_settings()``
+to initialize a ``RunSettings`` object. In this example, we use the Python
+executable to execute the application script, "script.py":
+
+.. code-block:: python
+
+    settings = exp.create_run_settings(exe="python", exe_args="script.py")
+
+We now have an instance of run settings name `settings`. Using the instance, instruct SmartSim
+to encapsulate the simulation as a Model through ``Experiment.create_model()``:
+
+.. code-block:: python
+
+    model = exp.create_model(name="example-model", run_settings=settings)
+
+Remember that all entities are launched, monitored and stopped by the Experiment instance.
+Specify the model instance, `model`, to ``Experiment.start()`` to start the standard model:
+
+.. code-block:: python
+
+    exp.start(model)
+
+When the Experiment Python driver script is executed, two files from the standard model will be created
+in the Experiment working directory:
+
+1. `model.out` : this file will hold outputs produced by the Model workload
```

Comment:
I believe the outputs are named using the model name. Do we want to have this say `example-model.out` or indicate `<model-name>.out`?

* Should we add an "outputs" section to the main Model documentation (not example) that documents the behavior so this could just be example-model.out?

Reply:
yes!

Reply:
to the adding an output section